python-backend/app/api/now_submissions/resources/application_resource.py

```diff
@@ -18,3 +27,42 @@ def get(self, application_guid):
             raise NotFound('Application not found')
 
         return application
+
+
+class ApplicationListResource(Resource, UserMixin, ErrorMixin):
+    @api.doc(
+        description='Get a list of applications. Order: receiveddate DESC',
+        params={
+            'page': f'The page number of paginated records to return. Default: {PAGE_DEFAULT}',
+            'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}',
+            'stuff': 'Comma-separated list of now submission applications to include in results.'
+        })
+    @requires_role_view_all
+    @api.marshal_with(PAGINATED_APPLICATION_LIST, code=200)
+    def get(self):
+        records, pagination_details = self._apply_filters_and_pagination(
+            page_number=request.args.get('page', PAGE_DEFAULT, type=int),
+            page_size=request.args.get('per_page', PER_PAGE_DEFAULT, type=int))
+
+        if not records:
+            raise BadRequest('Unable to fetch applications.')
```

Comment:
Are you intending to catch `None` here or an empty list? We likely don't want to throw a `BadRequest` for an empty list, since the request was likely fine -- just no records matched the parameters provided.

Reply:
Good point! I'll remove this